<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>std_conf</class>
 <widget class="QDialog" name="std_conf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="welkome_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome to School-Work</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="discription_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>First we need to add some configurations</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="max_YOA_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Maximal year of A.</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="max_YOA_spin_box">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="colums_count_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Colums count</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="colums_count_spin_box">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
